homeassistant/components/axis/config_flow.py

```diff
@@ -201,22 +203,42 @@ async def async_step_init(self, user_input=None):
         return await self.async_step_configure_stream()
 
     async def async_step_configure_stream(self, user_input=None):
-        """Manage the Axis device options."""
+        """Manage the Axis device stream options."""
         if user_input is not None:
             self.options.update(user_input)
             return self.async_create_entry(title="", data=self.options)
 
-        profiles = [DEFAULT_STREAM_PROFILE]
-        for profile in self.device.api.vapix.streaming_profiles:
-            profiles.append(profile.name)
+        schema = {}
+
+        # Stream profiles
```

Comment:
If it makes sense here, it would be more readable without all the `self.device.api.vapix` below 

```suggestion
        # Stream profiles
        vapix = self.device.api.vapix
```